src/components/EntityInfo/EntityMenu.vue

```diff
@@ -163,6 +163,48 @@
               </template>
               Submit a Consent to Continue Out of the province of B.C.
             </v-tooltip>
+
+            <!-- Request AGM Extension -->
+            <v-tooltip
+              right
+              content-class="right-tooltip"
+            >
+              <template #activator="{ on }">
+                <v-list-item
+                  v-if="enableAgmExtension"
+                  id="agm-ext-list-item"
+                  :disabled="!isAllowed(AllowableActions.AGM_EXTENSION)"
```

Comment:
Show as disabled (grayed out) if this action is not allowable (ie, Legal API checks some things for this entity such as state, etc).